lib/prometheus/business_collector.rb

```diff
@@ -92,6 +103,19 @@ def collect
     ).group('version').map(&:attributes).each do |item|
       @client_systems_by_os.observe(item['count'], version: item['version'])
     end
+
+    hosts_50_plus = PolicyHost.joins(:policy)
+                              .select('policies.account_id')
+                              .group(:policy_id, 'policies.account_id')
+                              .having('COUNT(policy_hosts.host_id) >= 50')
+
+    @total_accounts_with_huge_policies.observe(
+      Account.where(id: hosts_50_plus).distinct.count
```

Comment:
Accounts are not currently unique by account_number:

```suggestion
      Account.where(id: hosts_50_plus).count(:account_number)
```

Same comment below